### PR TITLE
Added bentley

### PR DESCRIPTION
**Pull request by ludoo0d0a**
Added the bentley cocktail ;)

### CODE REVIEW

File: src/recipes/bentley.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "name": "Bentley",
+    "description": "This coktail invented in 1920's in London, refers to the famous brand of luxe car. It is referenced in Savoy Cocktail Book from 1930. The legend says that the Bentley Boys (rich and famous british drivers) invented this drink after a win at 24h Le Mans.",
```
Comment (alfg) [inline src/recipes/bentley.json:3]:
```suggestion
    "description": "This cocktail invented in 1920's in London, refers to the famous brand of luxe car. It is referenced in Savoy Cocktail Book from 1930. The legend says that the Bentley Boys (rich and famous British drivers) invented this drink after a win at 24h Le Mans.",
```

File: src/recipes/bentley.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		"Pour ingredients in a shaker, add ice and shake.",
+        "Strain into an old fashioned glass.",
+        "Decorate with orange peel"
+    ],
+    "image": "bentley.jpg",
+        "keywords": [
+		"alcoholic",
+		"applejack",
+		"calvados",
+        "Dubonnet",
```
Comment (alfg) [inline src/recipes/bentley.json:33]:
```suggestion
        "dubonnet",
```

**Review by alfg — CHANGES_REQUESTED** [review]

**Comment by alfg** [comment]
Closing. Feel free to re-open with suggested fixes. Thanks!

**Comment by ludoo0d0a** [comment]
I cannot fix, in this PR because it's closed ; I will submit a new one with suggested fixes.
unless you reopen this PR :)

**Pull request closed without merging**